### ISSUE TITLE
Is this working with webGL and webCam?

### ISSUE THREAD

**Issue by PRO-Spirit** [cj-mills/barracuda-inference-yolox-demo#6]
Thanks your code, this is helpful for my personal project.
I want to know if this is working with webGL and webCam.

**Comment by cj-mills** [comment]
Hi @PRO-Spirit,

I recommend using the TensorFlow.js version of this project if you plan to use it with WebGL. The WebGL backend for Barracuda is buggy.

* [unity-tfjs-inference-yolox-demo](https://github.com/cj-mills/unity-tfjs-inference-yolox-demo)

Also, Barracuda was replaced with [Sentis](https://docs.unity3d.com/Packages/com.unity.sentis@2.1/manual/index.html), in case you didn't know.

* [sentis-inference-yolox-demo](https://github.com/cj-mills/sentis-inference-yolox-demo)

**Comment by PRO-Spirit** [comment]
Thank you for your reply.
I'm trying to make a rock-paper-scissors game. Is it possible to recognize
only 3 gestures when using this?

   - unity-tfjs-inference-yolox-demo
   <https://github.com/cj-mills/unity-tfjs-inference-yolox-demo>

rock, paper, scissors

Thank you!

**Comment by PRO-Spirit** [comment]
@cj-mills Thank you for your reply.
I'm trying to make a rock-paper-scissors game. Is it possible to recognize only 3 gestures when using this?
[unity-tfjs-inference-yolox-demo](https://github.com/cj-mills/unity-tfjs-inference-yolox-demo)
rock, paper, scissors

Thank you!

**Comment by cj-mills** [comment]
Hi @PRO-Spirit,

The processed object detections are stored in the [`bboxInforArray`](https://github.com/cj-mills/unity-tfjs-inference-yolox-demo/blob/59a46ace74aa4c5865b1a63504ea73b00f734cae/Assets/Scripts/InferenceController.cs#L169) variable on [this line](https://github.com/cj-mills/unity-tfjs-inference-yolox-demo/blob/59a46ace74aa4c5865b1a63504ea73b00f734cae/Assets/Scripts/InferenceController.cs#L169). This variable contains an array of [`BBox2DInfo`](https://christianjmills.com/posts/unity-bounding-box-2d-toolkit-walkthrough/#bbox2dinfo-struct) objects containing the filtered bounding boxes, class labels, and bounding box colors for each detected object. The list of class labels are loaded from a JSON colormap file like [this one](https://github.com/cj-mills/unity-tfjs-inference-yolox-demo/blob/main/Assets/Colormaps/hagrid-sample-250k-384p-colormap.json).

If you want to use the pre-trained model included with the demo, you can filter the detected objects based on the `label` values in the `bboxInforArray` variable.

Otherwise, you can train you own model using the tutorial series linked below:

* [Training YOLOX Models for Real-Time Object Detection in PyTorch](https://christianjmills.com/series/tutorials/pytorch-train-object-detector-yolox-series.html)
